tests/evaluate/test_twin_model.py

```diff
@@ -614,32 +614,32 @@ def test_raised_errors_with_tbrom_bad_name(self):
         try:
             twin._tbrom_resource_directory(rom_name="test")
         except TwinModelError as e:
-            assert "Twin model does not include any TBROM named" in str(e)
+            assert "Twin model does not include a TBROM named test." in str(e)
 
         try:
             twin.get_geometry_filepath(rom_name="test")
         except TwinModelError as e:
-            assert "Please call the geometry file getter with a valid TBROM name." in str(e)
+            assert "Call the geometry file getter with a valid TBROM" in str(e)
```

Comment:
Can a period be added?

Reply:
The purpose here is just to assert the right expression is raised, so it is not worth adding a period I think.